<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Swapna_Parsha" failures="3" tests="19" name="lms.LibraryManagementSystem.tests.LMSTest" time="54.281" errors="0" timestamp="2025-03-02T23:18:24 GMT" skipped="0">
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="AddBookWithBlankAuthorAndConfirm" time="2.775"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="AddBookWithBlankISBNAndConfirm" time="2.786"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="AddBookWithInvalidDataOrBlankSubmissionAndConfirm" time="2.678"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="AddBookAndConfirm" time="4.290"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="EditBookPageURLCheck" time="2.068"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="AddBookPageURLCheck" time="2.975"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="InvalidUserLoginFailErrorMessageValidation" time="2.012"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="AddBookWithInvalidDateAndConfirm" time="3.029">
    <failure message="expected [https://applicationforlibrarymanagementsystem.onrender.com/add-book] but found [https://applicationforlibrarymanagementsystem.onrender.com/books]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://applicationforlibrarymanagementsystem.onrender.com/add-book] but found [https://applicationforlibrarymanagementsystem.onrender.com/books]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at lms.LibraryManagementSystem.tests.LMSTest.AddBookWithInvalidDateAndConfirm(LMSTest.java:171)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- AddBookWithInvalidDateAndConfirm -->
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="editBookWithBlankTitleAndConfirm" time="2.225"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="AddBookWithBlankGenreAndConfirm" time="3.032"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="LogOutPageRedirectionValidation" time="4.288">
    <failure message="expected [https://applicationforlibrarymanagementsystem.onrender.com/login] but found [https://applicationforlibrarymanagementsystem.onrender.com/books]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://applicationforlibrarymanagementsystem.onrender.com/login] but found [https://applicationforlibrarymanagementsystem.onrender.com/books]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at lms.LibraryManagementSystem.tests.LMSTest.LogOutPageRedirectionValidation(LMSTest.java:322)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- LogOutPageRedirectionValidation -->
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="DeleteBookAndConfirm" time="2.815"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="DeleteAllBooksAndConfirmPageInfo" time="5.454">
    <failure message="expected [Page 1 of 1] but found [Page 1 of 0]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Page 1 of 1] but found [Page 1 of 0]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at lms.LibraryManagementSystem.pageobjects.BookListPage.assertPageInfo(BookListPage.java:111)
at lms.LibraryManagementSystem.tests.LMSTest.DeleteAllBooksAndConfirmPageInfo(LMSTest.java:274)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- DeleteAllBooksAndConfirmPageInfo -->
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="InvokeLoginPageFromTaskPageLink" time="1.731"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="AddBookWithBlankPriceAndConfirm" time="2.678"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="LoginWithBlankSubmissionErrorMessageValidation" time="1.927"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="editBookWithValidateAndConfirm" time="2.253"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="AddBookWithBlankDateAndConfirm" time="2.688"/>
  <testcase classname="lms.LibraryManagementSystem.tests.LMSTest" name="AddBookWithBlankTitleAndConfirm" time="2.577"/>
</testsuite> <!-- lms.LibraryManagementSystem.tests.LMSTest -->
